lib/membrane/live_compositor/live_compositor.ex

```diff
@@ -685,6 +674,13 @@ defmodule Membrane.LiveCompositor do
     {[notify_parent: {:output_registered, pad_ref, state.context}], state}
   end
 
+  @impl true
+  def handle_child_notification(:keyframe_request, {:output_processor, pad_id}, _ctx, state) do
```

Comment:
This is fine, but I would probably send `Request.KeyframeRequest` from the output processor